modules/ROOT/pages/portal-vanity-domain.adoc

```diff
@@ -65,11 +65,11 @@ For example:
 
 Add headers for each route described above.
```

Comment:
GLOBALLY avoid using "above" or "below."
See Style Ref.

Reply:
Fixed. Thank you!